template/en/default/mfa/page_requires_mfa.html.tmpl

```diff
@@ -0,0 +1,11 @@
+[% PROCESS "global/field-descs.none.tmpl" %]
+
+[% PROCESS global/header.html.tmpl
+   title  = "This page requires 2FA",
```

Comment:
This is slightly incorrect. "Your security group memberships require you to configure 2FA to continue using Bugzilla." or "Please configure 2FA to continue." is more accurate, assuming a hard-line stance anyways.